lib/kitchen/driver/oci.rb

```diff
@@ -237,7 +237,7 @@ def random_hostname(prefix)
 
       def random_password
         if instance_type == 'compute'
-          special_chars = %w[! " # & ( ) * + , - . /]
+          special_chars = %w[! " & ( ) * + , - . /]
         elsif instance_type == 'dbaas'
           special_chars = %w[# _ -]
```

Comment:
Is the # special character not an issue in DBaaS? as it only was an issue in Windows, and DBaaS doesnt run on Windows?

Reply:
That was my assumption, yes.  DBaaS doesn't run on Windows and to date, we haven't seen this issue on DBaaS